test/postControllerSuperTest.js

```diff
@@ -0,0 +1,42 @@
+/* global it, describe */
+const request = require('supertest');
+
+const app = require('../app');
```

Comment:
This is tough because this will include all controllers. Since you are only testing 1 controller, you probably want to create the express app here and just import the controller you are testing